### PR TITLE
[Editorial] Fix `browsing context` links.

### PR DESCRIPTION
**Pull request by mustaqahmed**
This fixes a part of #87.

***
<a href="https://pr-preview.s3.amazonaws.com/mustaqahmed/pointerlock/pull/88.html" title="Last updated on Apr 15, 2024, 12:08 AM UTC (4115bf0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerlock/88/79bbc6e...mustaqahmed:4115bf0.html" title="Last updated on Apr 15, 2024, 12:08 AM UTC (4115bf0)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,9 +205,9 @@ <h2>
           </p>
           <p>
             Pointer lock MUST fail unless the <a>target</a>'s
-            <a>shadow-including root</a> is the <a>active document</a> of a
-            [=Document/browsing context=] which is (or has an <a>ancestor
-            browsing context</a> which is) in focus by a window which is in
+            <a>shadow-including root</a> is the [=active document=] of a
+            [=Document/browsing context=] which is (or has an [=Document/ancestor
+            browsing context=] which is) in focus by a window which is in
```
Comment (marcoscaceres) [inline index.html]: While we are here 😄 

```suggestion
            [=Document/browsing context=] that is (or has an [=Document/ancestor
            browsing context=] that is) in focus by a window that is in
```

**Review by marcoscaceres — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -205,9 +205,9 @@ <h2>
           </p>
           <p>
             Pointer lock MUST fail unless the <a>target</a>'s
-            <a>shadow-including root</a> is the <a>active document</a> of a
-            [=Document/browsing context=] which is (or has an <a>ancestor
-            browsing context</a> which is) in focus by a window which is in
+            <a>shadow-including root</a> is the [=active document=] of a
```
Comment (mustaqahmed) [inline index.html]: @marcoscaceres Any clue why ReSpec is still not happy with this PR (see "Generate static HTML" step [here](https://github.com/w3c/pointerlock/actions/runs/7713337739/job/21022995725?pr=88))?  Perhaps the symbols are not exposed in the source specs?  Perhaps I need to "include" other spec definitions here somehow?

**Comment by marcoscaceres** [comment]
Oh, looks like it's sad about "active document" and some other things.

File: index.html (hunk 2 above)
Comment (marcoscaceres) [inline index.html]:
```suggestion
            <a>shadow-including root</a> is the [=navigable/active document=] of a
```

**Comment by marcoscaceres** [comment]
There were two instances of active document, so fixed them here: https://github.com/w3c/pointerlock/commit/9e39c46e1bc2e03b32125e2b00f788a2a966a5bd

**Pull request closed without merging**